src/sbt-test/sbt-plugins/simple/docker/build.sbt

```diff
@@ -1,8 +1,8 @@
 // Map `foo` in the project root to `bar` in the docker image.
-dockerCopyMappings += (baseDirectory.value / "foo", "bar")
+dockerCopyMappings += (file("foo"), "bar")
 
 // Map src/main/sourcey to `sourcey` in the docker image.
-dockerCopyMappings += (file("sourcey"), "sourcey")
+dockerCopyMappings += (sourceDirectory.value / "main" / "sourcey", "sourcey")
```

Comment:
That's not part of the public API. This is in integration test of the public API.